drivers/i2c/i2c_ll_stm32_v1.c

```diff
@@ -28,6 +28,18 @@ LOG_MODULE_REGISTER(i2c_ll_stm32_v1);
 #define I2C_REQUEST_READ        0x01
 #define HEADER                  0xF0
 
+static void ll_i2c_generate_start_condition(I2C_TypeDef *i2c)
```

Comment:
Let it be `stm32_i2c_generate_start_condition` and reserve `ll` namespace for pure LL API abstraction when needed.